test/fixtures/EditionFactoryFixture.sol

```diff
@@ -21,10 +21,52 @@ library EditionDataWither {
         self.creatorAddr = creatorAddr;
         return self;
     }
+
+    function withMinterAddr(EditionData memory self, address minterAddr) internal pure returns (EditionData memory) {
+        self.minterAddr = minterAddr;
+        return self;
+    }
```

Comment:
the buildoooooooorrrrrrrr!!!!!!

Reply:
haha ngl it's pretty convenient to use in tests :)